lib/driver.js

```diff
@@ -190,12 +190,14 @@ class AndroidUiautomator2Driver extends BaseDriver {
         // find and copy, or download and unzip an app url or path
         this.opts.app = await this.helpers.configureApp(this.opts.app, [APK_EXTENSION, APKS_EXTENSION]);
         await this.checkAppPresent();
-      } else if (this.appOnDevice) {
+      } else if (this.opts.appPackage) {
```

Comment:
Does this driver now take the `appPackage` from the `app` capability if it is there? Why not continue using `this.appOnDevice`?

Reply:
`appOnDevice` was a blind copy paste from AndroidDriver, which is not even assigned anywhere in UIA2

Reply:
> Does this driver now take the appPackage from the app capability if it is there?

It was always like that:

```
    // get appPackage et al from manifest if necessary
    const appInfo = await helpers.getLaunchInfo(this.adb, this.opts);
    // and get it onto our 'opts' object so we use it from now on
    Object.assign(this.opts, appInfo || {});
```

Reply:
Ah. Getters don't get assigned like functions do. Alas.